dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/dataset/DatasetFieldsInitializerTest.java

```diff
@@ -224,16 +225,20 @@ public void updateDatasetFieldIncludeFlag() {
                 .thenReturn(prepareHiddenFields(Lists.newArrayList(titleField.get())));
 
         //when
-        List<DatasetField> updatedDsf = datasetFieldsInitializer.updateDatasetFieldIncludeFlag(datasetFields, dataverse);
+        Map<MetadataBlock, List<DatasetFieldsByType>> updatedDsf = datasetFieldsInitializer.groupAndUpdateFlagsForEdit(datasetFields, dataverse);
 
-        Optional<DatasetField> titleDsf = updatedDsf.stream()
-                .filter(datasetField -> datasetField.getDatasetFieldType().getName().equals("title"))
+        Optional<DatasetFieldsByType> titleFields = updatedDsf.values().stream()
+                .flatMap(fieldsByTypes -> fieldsByTypes.stream())
+                .filter(fieldByType -> fieldByType.getDatasetFieldType().getName().equals("title"))
                 .findAny();
 
         //then
-        Assert.assertEquals(3, updatedDsf.stream().filter(DatasetField::isInclude).count());
-        Assert.assertTrue(titleDsf.isPresent());
-        Assert.assertFalse(titleDsf.get().isInclude());
+        Assert.assertEquals(1, updatedDsf.values().stream()
```

Comment:
You can also use `Assertions.assertAll(() ->,...)` so it will validate all assertions at once without braking on the first one.